osu.Game/Graphics/UserInterface/ISelectable.cs

```diff
@@ -0,0 +1,10 @@
+// Copyright (c) ppy Pty Ltd <contact@ppy.sh>. Licensed under the MIT Licence.
+// See the LICENCE file in the repository root for full licence text.
+
+namespace osu.Game.Graphics.UserInterface
+{
+    public interface ISelectable
+    {
+        bool Selected { get; set; }
```

Comment:
Why is this not an `Bindable<bool>` if every usage is consuming as a bindable?

Alternative, if you want to avoid using a bindable here, why not move all the "on selected" code inside the property `Set` methods?

Reply:
I'm even more confused now. You made this a non-bindable by using `IStateful` and as a result the class has this weird mapping dictionary. Is there a reason you didn't take my advice and went in a completely different direction?

Reply:
The mapping dictionary is used so that the events can be removed properly if the drawable is removed from this container. (so that items moved to another container don't trigger updates) 

If it uses bindables, then I'd need to keep a list of bindables and perform roughly the same thing.

Is that something that we don't need to consider, since we aren't expecting items to be removed from this container.

Reply:
I think either way this should be simplified to not require unbinding the event. Probably just disallow removing items from the container and call it a day in that case?

The benefit of bindables is that it takes any guess work out of event binding, but if there's no `Remove` method then there should be no issue in the first place (due to the container basically holding a lifetime on the items added).